services/retrieve_and_rank/v1.js

```diff
@@ -292,7 +301,7 @@ module.exports = function (RED) {
               //Note. temporary workaround until bug is fixed in Node SDK
               //Stream zip file from watson api to temp directory,
               //read from temp directory and pass on in msg.payload as buffer
-              var url = "https://gateway.watsonplatform.net/retrieve-and-rank/api/v1/solr_clusters/"+params.cluster_id+"/config/"+params.config_name;
+              var url = 'https://gateway.watsonplatform.net/retrieve-and-rank/api/v1/solr_clusters/'+params.cluster_id+'/config/'+params.config_name;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/BetaWorks-NodeRED-Watson/node-red-node-watson/file/2690663401/issues/source?bid=3330694&fileBranchId=3336646#l304)